README.md

```
@@ -709,6 +729,93 @@ but real-world tuning needs an actual malware corpus with known
+
+```bash
+sudo insmod av/av.ko
+cd userspace/avd && make && cd ../..
+sudo userspace/avd/avd rules corpus/fuzzy_hashes.txt /var/lib/av-quarantine corpus/tlsh_hashes.txt
+
+# in another terminal
+/tmp/ptrace_test_variant
+```
+
+Expect in `avd`'s output:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)